internal/server/statvar/golden/get_statvar_match_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package golden
+
+import (
+	"context"
+	"path"
+	"runtime"
+	"testing"
+
+	pb "github.com/datacommonsorg/mixer/internal/proto"
+	"github.com/datacommonsorg/mixer/test/e2e"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/protobuf/testing/protocmp"
+)
+
+func TestGetStatVarMatch(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+
+	_, filename, _, _ := runtime.Caller(0)
+	goldenPath := path.Join(path.Dir(filename), "get_statvar_match")
+
+	testSuite := func(mixer pb.MixerClient, recon pb.ReconClient, latencyTest bool) {
+		for _, c := range []struct {
+			propertyValues map[string]string
+			goldenFile     string
+		}{
+			{
+				map[string]string{
+					"gender":           "Female",
+					"measuredProperty": "count",
+					"nativity":         "USC_ForeignBorn",
+				},
+				"female_usc_foreignborn.json",
```

Comment:
These are pretty large files for testing. Can we trim it down to managable size, say at most 50 statvars?